.travis.yml

```diff
@@ -17,11 +20,14 @@ cache:
 before_install:
   - mkdir -p $HOME/.local/bin
   - export PATH=$HOME/.local/bin:$PATH
-  #- export STACK_VERSION=0.1.6.0
-  #- export STACK_FULLVERSION=stack-$STACK_VERSION-linux-x86_64
-  #- travis_retry curl -L https://github.com/commercialhaskell/stack/releases/download/v$STACK_VERSION/$STACK_FULLVERSION.tar.gz | tar -xz -C /tmp $STACK_FULLVERSION/stack
-  #- mv /tmp/$STACK_FULLVERSION/stack $HOME/.local/bin
-  - travis_retry curl -L https://github.com/rubik/argon/releases/download/test-stack-1.7/stack-linux-x64.tar.bz2 | tar -xj -C $HOME/.local/bin
+  - export STACK_FULLVERSION=stack-$STACK_VERSION-linux-x86_64
+  - if [ $STACK_VERSION == "0.1.7.0" ]; then
+        curl -L https://github.com/rubik/argon/releases/download/test-stack-1.7/stack-linux-x64.tar.bz2 | tar -xj -C $HOME/.local/bin;
+    else
+        curl -L https://github.com/commercialhaskell/stack/releases/download/v$STACK_VERSION/$STACK_FULLVERSION.tar.gz | tar -xz -C /var/tmp;
+        mv /var/tmp/$STACK_FULLVERSION/stack $HOME/.local/bin;
```

Comment:
The 1.8.0 release has docs as well:

```bash
$ tar -tf stack-0.1.8.0-linux-x86_64.tar.gz 
stack-0.1.8.0-linux-x86_64/stack
stack-0.1.8.0-linux-x86_64/CONTRIBUTING.md
stack-0.1.8.0-linux-x86_64/ChangeLog.md
stack-0.1.8.0-linux-x86_64/GUIDE.md
stack-0.1.8.0-linux-x86_64/LICENSE
stack-0.1.8.0-linux-x86_64/README.md
stack-0.1.8.0-linux-x86_64/doc/GUIDE.md
stack-0.1.8.0-linux-x86_64/doc/MAINTAINER_GUIDE.md
stack-0.1.8.0-linux-x86_64/doc/SIGNING_KEY.md
stack-0.1.8.0-linux-x86_64/doc/architecture.md
stack-0.1.8.0-linux-x86_64/doc/build_command.md
stack-0.1.8.0-linux-x86_64/doc/dependency_visualization.md
stack-0.1.8.0-linux-x86_64/doc/docker_integration.md
stack-0.1.8.0-linux-x86_64/doc/faq.md
stack-0.1.8.0-linux-x86_64/doc/install_and_upgrade.md
stack-0.1.8.0-linux-x86_64/doc/nonstandard_project_init.md
stack-0.1.8.0-linux-x86_64/doc/shell_autocompletion.md
stack-0.1.8.0-linux-x86_64/doc/stack_and_ghcjs.md
stack-0.1.8.0-linux-x86_64/doc/yaml_configuration.md
```
It also has a path prefix when extracted by default, I can update to: 

```bash
curl -L https://github.com/commercialhaskell/stack/releases/download/v$STACK_VERSION/$STACK_FULLVERSION.tar.gz |\
 tar xz --strip-components 1 -C ~/.local/bin $STACK_FULLVERSION/stack
```
Which will just extract and store **stack** directly in **~/local/bin**?